Instructions.txt

```diff
@@ -0,0 +1,132 @@
+########################################
+### Prepare system (Linux Mint 18.1) ###
+########################################
+
+### Step 1: Create Project Directory
+mkdir TKproj
+cd TKproj
+
+
+### Step 2: Clone sources from github
+git clone https://github.com/kevinhughes27/TensorKart.git
+git clone https://github.com/kevinhughes27/mupen64plus-input-bot.git
+git clone https://github.com/mupen64plus/mupen64plus-core.git
+git clone https://github.com/mupen64plus/mupen64plus-input-sdl.git
+
+
+### Step 3: Install dependancies
+sudo apt-get install dpkg-dev build-essential swig python2.7-dev libwebkitgtk-dev libjpeg-dev libtiff-dev checkinstall ubuntu-restricted-extras freeglut3 freeglut3-dev libgtk2.0-dev  libsdl1.2-dev libgstreamer-plugins-base0.10-dev 
+
+sudo apt install python-pip
+sudo pip install --upgrade pip
+sudo pip install -U pip setuptools
```

Comment:
I think that was related to wx again... I read many times I needed to make sure pip and setuptools were updated, eg here: https://wiki.wxpython.org/How%20to%20install%20wxPython

Once I got wx working, I didn't try to reduce the number of libraries used. It is possible this step can be left out.